lib/Conversion/TosaToTTIR/TosaToTTIRPass.cpp

```diff
@@ -22,7 +22,7 @@
 #include "mlir/Transforms/DialectConversion.h"
 
 using namespace mlir;
-using namespace tt;
+using namespace mlir::tt;
```

Comment:
You removed a lot of code from this file, are all of the above includes still required?

Reply:
If clang is properly set up, it grays out unused imports. Try removing them and see if compile and a couple of tests will work.

Reply:
I have removed the unused includes.

Reply:
Thanks!